cirq/protocols/json_test_data/Result.json

```diff
@@ -1,6 +1,6 @@
 [
```

Comment:
for backwards compatibility testing, the old TrialResult.json should be left *unmodified* and a new Result.json should be added